.travis.yml

```diff
@@ -1,37 +1,30 @@
 sudo: required
 dist: trusty
-
-
-language: java
-python:
-        - "2.7"
 language: go
+python:
+- '2.7'
 go:
-        - 1.8.1
+- 1.8.1
 go_import_path: github.com/GoogleCloudPlatform/runtimes-common
-
 jdk:
-  - oraclejdk8
-
+- oraclejdk8
 env:
-  - NODE_VERSION="6.11.4"
-
+- NODE_VERSION="6.11.4"
 before_install:
-  - pip install --user --upgrade -r requirements.txt
-  - nvm install $NODE_VERSION
-
+- openssl aes-256-cbc -K $encrypted_de21ef459881_key -iv $encrypted_de21ef459881_iv
```

Comment:
where does `$encrypted_de21ef459881_key` come from?

Reply:
These secrets get created when you run travis encrypt command.
The formatting is also done by the same command.
https://docs.travis-ci.com/user/encryption-keys/